train.py

```diff
@@ -168,18 +182,22 @@ def on_validation_end(self, trainer, pl_module):
 
     # TRAINING PARAMETERS
 
-    parser.add_argument('--epochs', dest='epochs', type=int, default=40)
+    parser.add_argument('--epochs', dest='epochs', type=int, default=20)
     parser.add_argument('--patience', dest='patience', type=int, default=10)
-    parser.add_argument('--batch-size', dest='batch_size', type=int, default=64)
-    parser.add_argument('--lr', dest='l_rate', type=float, default=1e-4)
-    parser.add_argument("--min-lr", dest='minimum_lr', type=float, default=1e-5, help="Minimum Learning Rate")
-    parser.add_argument("--lr-decay", dest='lr_decay', type=float, default=1e-3, help="Learning rate (weight) decay")
+    #batch_size 1 for graph networks
+    # parser.add_argument('--batch-size', dest='batch_size', type=int, default=64)
```

Comment:
This should be commented out, we can just set it to 1 if we want to work with GNNs

Reply:
Changed it